### DISCUSSION TITLE
selector in CSS?

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#53] [Q&A] [answered]
What is the use of a class selector in CSS?

**Comment by Bunny77K9** [comment, marked as answer]
Class selectors in CSS begin with a “.” (period) key and are followed by the name of the class. It is used to select a statement and modify the style of that element in the corresponding part of the HTML tag.

Consider the following example:

.exampleclass {font-family: TimesNewRomanl; font-size: 20; background: red;}
<div class="sampleclass">
...
</div>

Next up on these web technologies interview questions, let us understand a little about the use of Webkit.
